omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
the problem with that was that we are passing a variable from gsp. so even if i have an init like
$scope.init = function(uuid){
$scope.personUuid = uuid;
}
and then call a function 
function loadTags(){
}
the parameter does not pass when loadTags is called for this first time.